### DISCUSSION TITLE
Support for Matter

### DISCUSSION THREAD

**Discussion by loozhengyuan** [stm32duino/Arduino_Core_STM32#2573] [Ideas]
ST currently provides [Matter](https://csa-iot.org/all-solutions/matter/) support for STM32 MCUs through [X-CUBE-MATTER](https://www.st.com/en/embedded-software/x-cube-matter.html). It will be nice to see support for this in the Arduino core for STM32.

Not sure if it is useful for ideas/inspiration but Silicon Labs collaborated with Arduino early this year to publish their [Arduino Matter library](https://github.com/SiliconLabs/arduino/tree/main/libraries/Matter) and Espressif has recently published an initial version of their [Matter library](https://github.com/espressif/arduino-esp32/pull/10467) for their Arduino core for ESP32.

**Comment by fpistm** [comment]
Hi @loozhengyuan 

Thanks for this ideas. I convert it as a discussion as it it not really linked to a core issue.
